### ISSUE TITLE
Line was eaten on merge

### ISSUE THREAD

**Issue by ayg** [MatmaRex/mediawikireleasenotes-driver#2]
When I tried rebasing my LanguageFactory branch on top of my Namespace branch, a line from RELEASE-NOTES was eaten.  The commits are on Gerrit:

https://gerrit.wikimedia.org/g/mediawiki/core/+/14aecc4f99921f7b521394293bfc5e33a595fa7e
https://gerrit.wikimedia.org/g/mediawiki/core/+/dae9789009f3aba6a2de2c79b1bf946b38fffe7d

Of the two added lines

+* (Almost) all Language static methods are now deprecated.  Use LanguageFactory
+  methods instead.

the second one was lost.

**Comment by MatmaRex** [comment]
I can reproduce this. Commands for future reference:
```
git fetch origin dae9789009f3aba6a2de2c79b1bf946b38fffe7d
git checkout dae9789009f3aba6a2de2c79b1bf946b38fffe7d -b namespace
git fetch origin 14aecc4f99921f7b521394293bfc5e33a595fa7e
git checkout 14aecc4f99921f7b521394293bfc5e33a595fa7e -b languagefactory
git rebase namespace
```

This happens because the left-side file already has a line containing exactly `  methods instead.` (a few lines above the one you added), so the line `added = sright[i] - sparent[i]` (this calculates a set difference) doesn't return it.

The script should probably compare "list items", somehow, rather than lines.

**Comment by MatmaRex** [comment]
@ayg Thank you. Fixed in 560c589e19db957bec75d1020853a44c0028d4af. However, I feel like my solution is more complicated than it should be… Improvements are welcome :)
